internal/pkg/manifest/rd_web_svc.go

```diff
@@ -57,6 +58,9 @@ func NewRequestDrivenWebService(props *RequestDrivenWebServiceProps) *RequestDri
 	svc.RequestDrivenWebServiceConfig.ImageConfig.Image.Location = stringP(props.Image)
 	svc.RequestDrivenWebServiceConfig.ImageConfig.Build.BuildArgs.Dockerfile = stringP(props.Dockerfile)
 	svc.RequestDrivenWebServiceConfig.ImageConfig.Port = aws.Uint16(props.Port)
+	if props.Platform != nil {
+		svc.RequestDrivenWebServiceConfig.InstanceConfig.Platform = props.Platform
+	}
```

Comment:
I'm trying to remember why we even allow instance config to have a Platform for RDWS? since it only supports linux arm64

Reply:
Took it out of `RequestDrivenWebServiceProps` bc it won't go in manifests, but don't we need it in InstanceConfig for BuildArgs?

Reply:
Won't we need this field if `ARM` is redirecting to `amd64` for `BuildArgs`?